### DISCUSSION TITLE
https://www.xiaogenban1993.com/blog/24.08/LLM%E5%BA%94%E7%94%A8%E6%A6%82%E8%BF%B0

### DISCUSSION THREAD

**Discussion by giscus** [sunwu51/notebook#68] [General]
# https://www.xiaogenban1993.com/blog/24.08/LLM%E5%BA%94%E7%94%A8%E6%A6%82%E8%BF%B0

Generated by FreeCoder

https://www.xiaogenban1993.com/blog/24.08/LLM%E5%BA%94%E7%94%A8%E6%A6%82%E8%BF%B0

**Comment by mabangwang** [comment]
hello 博主 Llama 3.3 70B当时你下载了多久

**Comment by anansite** [comment]
邮件已收到，我会尽快查看
